### DISCUSSION TITLE
How to fully leverage Wolverine with its HTTP package

### DISCUSSION THREAD

**Discussion by giscus** [timdeschryver/timdeschryver.dev#282] [Announcements]
# How to fully leverage Wolverine with its HTTP package

Taking the next step to keep the architecture and the codebase simple with Wolverine. In this blog post, we'll take a look at the WolverineFx.HTTP package to make the interaction with HTTP requests easier.

https://timdeschryver.dev/blog/how-to-fully-leverage-wolverine-with-its-http-package

**Comment by jeremydmiller** [comment]
Hey Tim, I really appreciate these posts. If you wanted, the last signature could be:

```
    [WolverinePost("carts"), EmptyResponse]
    public static CartCreated Create(CreateCart command, ShoppingCartDbContext context)
    {
        var cart = new Cart(command.Id);
        context.Add(cart);
 
        return new CartCreated(cart.Id);
    }

```

and that would give you the same behavior and OpenAPI metadata

**Reply by timdeschryver** [reply]
Thanks @jeremydmiller, I've also included that example.
